src/firefly/js/util/Logger.js

```diff
@@ -122,16 +124,19 @@ function log(msg, {level=2, tag=''}) {
     if (debugLevel() >=  level) {
         const doLog = debugTags().length === 0 || find(debugTags(), (t) => !!tag.match(t));
         if (doLog) {
+            const now = new Date();
+            const ts = '%c' + sprintf('%2s:%2d:%2d.%3d ', now.getHours(), now.getMinutes(), now.getSeconds(), now.getMilliseconds());
+
+            tag = tag ? tag + ':' : ':';
             if (typeof msg === 'object') {
-                console.log(tag ? tag + ': ==>' : '');
-            } else {
-                msg = tag ? `${tag}: ${msg}` : msg;
+                console.log(ts, 'color:maroon', tag, '==>');
             }
-            if (level === 0) console.error(msg);
-            if (level === 1) console.warn(msg);
-            if (level === 2) console.info(msg);
-            if (level === 3) console.debug(msg);
-            if (level === 4) console.trace(msg);
+
+            if (level === 0) console.error(ts, 'color:maroon', tag, msg);
+            if (level === 1) console.warn(ts, 'color:maroon', tag, msg);
+            if (level === 2) console.info(ts, 'color:maroon', tag, msg);
+            if (level === 3) console.debug(ts, 'color:maroon', tag, msg);
+            if (level === 4) console.trace(ts, 'color:maroon', tag, msg);
```

Comment:
The color is only for timestamp.  This makes it easier to identify the message.
When you console.error or console.warn, it will be in red and yellow by default with icons.